lib/isaac/isaac_version.hpp

```diff
@@ -1,8 +1,8 @@
 #pragma once
 
 #define ISAAC_VERSION_MAJOR 1
-#define ISAAC_VERSION_MINOR 6
-#define ISAAC_VERSION_PATCH 0
+#define ISAAC_VERSION_MINOR 5
+#define ISAAC_VERSION_PATCH 2
```

Comment:
@psychocoderHPC I think, here went something wrong - shouldn't that change from 1.5.2 to 1.6.0 - but seems to do the inverse

Reply:
Yes, may be a mistake during rebase with wrong version used

Reply:
fixed